crates/fluvio-cli/src/topic/list.rs

```diff
@@ -103,16 +103,19 @@ mod display {
                 .iter()
                 .map(|metadata| -> Row {
                     let topic = &metadata.spec;
-                    row![
-                        l -> metadata.name,
-                        c -> topic.type_label(),
-                        c -> topic.partitions_display(),
-                        c -> topic.replication_factor_display(),
-                        c -> format_duration(Duration::from_secs(topic.retention_secs() as u64)),
-                        c -> topic.get_compression_type(),
-                        c -> metadata.status.resolution.to_string(),
-                        l -> metadata.status.reason
-                    ]
+
+                    Row::from([
+                        Cell::new(metadata.name.to_string()),
+                        Cell::new(topic.type_label()),
+                        Cell::new(topic.partitions_display()),
```

Comment:
could you change this with
```suggestion
                        Cell::new(topic.partitions_display()).set_alignment(CellAlignment::Left);,
```
in order to be similar to previous output, the same applies for the other columns of the row.


Compare `fluvio topic list` in the stable version with the changes of this PR